### PR TITLE
Sets

### PR DESCRIPTION
**Pull request by MARY2030**
Homework Sets

### CODE REVIEW

File: src/Double1.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        Double obj1 = new  Double(1);
+
+        Double obj2 = new  Double(1);
+
+
+        Double obj3 = new  Double(2);
+
+
+        Double obj4 = new  Double(2);
```
Comment (mobilman89) [inline src/Double1.java]: It is not duplicates! it is a different objects. You should have create new Set<Integer> and tried to add two the same values

File: src/Map.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+//Initiate a Set. Using For loop add integers from 1 to 10
+//Remove all odd numbers from sets.
+
+public class Map {
+    public static void main(String[] main) {
+
+        Set<Integer> setNum = new HashSet<>();
+
+        for (int i = 1; i <= 10; i++) {
```
Comment (mobilman89) [inline src/Map.java]: You should have added all namubers from 1 to 10 using loop. and then delete all odds using another loop.

File: src/Map.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            setNum.add(i);
+            System.out.println(i);
+
+            if (i % 2 == 0) {
```
Comment (mobilman89) [inline src/Map.java]: you remove even numbers here but should remove odds

File: src/Double1.java (hunk 1 above)
Comment (MARY2030) [reply src/Double1.java]: hello, by definition A set cannot have duplicate elements by its mere definition. The correct structure to allow duplicate elements .I have thought to. create two different set and combine, but I followed the definition. that is why I created 2 different objects. distraction error for the big capital letter on variable.it only apples for classes.the odds number I %2!=0

**Pull request merged**